packages/iris-grid/src/IrisGridUtils.js

```diff
@@ -166,8 +168,8 @@ class IrisGridUtils {
       selectedSearchColumns,
       invertSearchColumns = true,
       pendingDataMap = [],
+      frozenColumns = [],
```

Comment:
Shouldn't initialize to an empty array, we don't want to save frozenColumns unless user has set them